.github/workflows/test-go.yaml

```diff
@@ -17,6 +17,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         go-version: ['^1.17.0']
+        mysql: ["mysql:5.7", "mysql:8"]
```

Comment:
So this means that we'll spend twice as much time in CI for this particular part. Do we want that? Should we instead make this a nightly cron or something?

I'm ok with merging it as is until we do.

Reply:
The checks actually run in parallel by default when you use a matrix strategy. Note that there is a concurrency limit (currently 20) but I'm looking into upgrading us to a paid plan to get the limit bumped (to 60).

Reply:
Correct, they are two jobs that run in parallel.

<img width="715" alt="Screen Shot 2022-01-11 at 21 23 07" src="https://user-images.githubusercontent.com/2073526/149042117-1a887fff-ad2e-4f6f-a784-fcd7f1ee132d.png">